tests/integration/conftest.py

```diff
@@ -1306,7 +1306,7 @@ def mock_task_manager(request, persistent, storage, temp_storage, get_cred, mock
     Mock celery task manager for testing
     """
     _ = mock_app_callbacks
-    if "disable_task_manager_mock" in request.keywords:
+    if request.module.__name__ == "test_task_manager":
```

Comment:
Checking `request.keywords` does not work when the fixture has a module scope, so the tests that required this mock to be skipped did not actually have the mock skipped. Checking for module name instead (only tests in the test_task_manager integration test script runs jobs in an actual celery worker process and need this mock to be skipped)

Reply:
Good to find this. I encountered some similar failures locally previously. 👍